### DISCUSSION TITLE
port exposure / port forwarding ? expose app from Docker to Host / Host network ?

### DISCUSSION THREAD

**Discussion by Y3NK** [coder/coder#15912] [General]
Hello, I tried to expose a port; running in docker in port 8080

I don't see the web ui to create a port FW, have you any idea why ?

I tried to create an app:
```tf
resource "coder_app" "port-fw" {
  agent_id     = coder_agent.main.id
  slug         = "port-fw"
  display_name = "port-fw"
  url          = "http://localhost:8080"
  # icon         = "/icon/code.svg"
  subdomain    = true
  share        = "authenticated"
 }
 ``` 
  
but logo greyed out and impossible to start.
  
How can I let the developer to allow him to choose to add another port ? (let him build whaterver front-end ) which need to be exposed and accessible from Host / Host network ?

[EDIT:] I am exposing coder behind Caddy, whose listen 443 and get a proper SSL cert
 
 Thanks

**Comment by matifali** [comment]
Have you configured a [`wid-card-access-url`](https://coder.com/docs/admin/setup#wildcard-access-url)?
